content/git/concepts/diff/diff.md

```diff
@@ -0,0 +1,127 @@
+
+ARIANEA
+How long?
+
+COBBYTO
+According to my calculations, the time span is approximately three days at the top layer, three months one layer down, and six years a level after.
+
+ARIANEA
+Who would want to live in an illusory world for six years?
+```
+
+Using `git diff` in the terminal will produce the following output:
+
+```bash
+$ git diff
+diff --git a/scene.txt b/scene.txt
+index c16c37f..c680bb4 100644
+--- a/scene.txt
++++ b/scene.txt
+@@ -2,4 +2,10 @@ COBBYTO
+ In a dream, brain activity will be roughly thirty times higher than usual. The effect is increased by entering a dream within that dream.
+ 
+ ARIANEA
+-How long?
+\ No newline at end of file
++How long?
++
++COBBYTO
++According to my calculations, the time span is approximately three days at the top layer, three months one layer down, and six years a level after.
++
++ARIANEA
++Who would want to live in an illusory world for six years?
+\ No newline at end of file
+```
+
+With only one file tracked, using  `git diff` is the same as using  `git diff scene.txt`
+
+### 2. git diff --cached
+
+`git diff --cached` displays the changes between the staging area and the last commit (i.e changes between what is [staged](https://www.codecademy.com/resources/docs/git/add) but not yet committed).
+
+If `git diff --cached` is run in the terminal, no output is produced. This is expected because nothing has been added to the staging area yet. Therefore, there's no status change between the latter and the last commit. This can be verified by running [`git status`](https://www.codecademy.com/resources/docs/git/status).
+
+```bash
+$ git status
+On branch master
+Changes not staged for commit:
+  (use "git add <file>..." to update what will be committed)
+  (use "git checkout -- <file>..." to discard changes in working directory)
+
+        modified:   scene.txt
+
+no changes added to commit (use "git add" and/or "git commit -a")
+```
+
+Adding `scene.txt` to the staging area and running `git diff --cached` in the terminal should yield the following:
+
+```bash
+$ git diff --cached
+diff --git a/scene.txt b/scene.txt
+index c16c37f..c680bb4 100644
+--- a/scene.txt
++++ b/scene.txt
+@@ -2,4 +2,10 @@ COBBYTO
+ In a dream, brain activity will be roughly thirty times higher than usual. The effect is increased by entering a dream within that dream.
+ 
+ ARIANEA
+-How long?
+\ No newline at end of file
++How long?
++
++COBBYTO
++According to my calculations, the time span is approximately three days at the top layer, three months one layer down, and six years a level after.
++
++ARIANEA
++Who would want to live in an illusory world for six years?
+\ No newline at end of file
+```
+
+A quick look at the output shows it is the same as the one from the previous _1. git diff_ section, since the changes saved have been added to the staging area. Conversely, if `git diff` is run in the terminal, no output is produced.
```

Comment:
```suggestion
The output is the same as the output for `git diff` in the previous section because the changes saved have been added to the staging area. Conversely, if `git diff` is run in the terminal, no output is produced.
```